### PR TITLE
Leap

### LINKED ISSUE #475: add Leap
### Some reference to the SDK, such as the developer site or a wikipedia article.

https://developer-archive.leapmotion.com/documentation/csharp/devguide/Leap_SDK_Overview.html

### SteamDB links

https://steamdb.info/app/623810/

### Possible way to detect it

Leap.dll/LeapC.dll

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
### SteamDB app page links to a few games using this
https://steamdb.info/app/223750/
https://steamdb.info/app/274920/
https://steamdb.info/app/457550/
https://steamdb.info/app/623810/
https://steamdb.info/app/766300/
https://steamdb.info/app/871170/
https://steamdb.info/app/966180/
https://steamdb.info/app/1364390/
https://steamdb.info/app/1435710/
https://steamdb.info/app/1913200/
https://steamdb.info/app/1957330/

### Brief explanation of the change
Add Leap Motion SDK. Closes #475.

### CODE REVIEW

File: descriptions/SDK.Leap.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The Leap Motion SDK is a controller-free hand tracking and haptics software by [**Ultraleap**](https://www.ultraleap.com/)
```
Comment (Lifeismana) [inline descriptions/SDK.Leap.md:1]:
```suggestion
[Leap Motion SDK](https://developer.leapmotion.com/) is a controller-free hand tracking and haptics software by Ultraleap
```

**Review by Lifeismana — CHANGES_REQUESTED** [review]
needs some non matching tests

**Comment by MichaelHinrichs** [comment]
> needs some non matching tests

https://github.com/SteamDatabase/FileDetectionRuleSets/pull/496/commits/7aa24821be8b1a11dcac0123bb2071be5bd3ebe8
How's that?

**Comment by Lifeismana** [comment]
you don't have to put swear words in your commits

**Comment by MichaelHinrichs** [comment]
Done.

**Comment by Lifeismana** [comment]
merged in 6318ab370aa22de152b039381b153497b87600b4

**Pull request closed without merging**